### ISSUE TITLE
Technical Highlight: Fantastic Feathers Features

### ISSUE THREAD

**Issue by marshallswain** [canjs/canjs#2838]
Fantastic Feathers Features could be included as a technical highlight.  Official support for what's probably the fastest growing Node.js API framework is a pretty standout feature, in my opinion.  We'd for sure want to mention that it works with any framework, but we have some nice tools for working with Feathers.

@chasenlehara @justinbmeyer

**Comment by justinbmeyer** [comment]
Agreed ... but lets wait on this until Bitcentive proves that they are working together fine.  I'd probably have a section around 3rd party integrations ... Kefier would be another sub-point.
